packages/shared/src/components/widgets/SocialShareList.tsx

```diff
@@ -54,59 +65,52 @@ export function SocialShareList({
         />
       )}
       <SocialShareButton
-        href={getTwitterShareLink(link, description)}
         icon={<TwitterIcon />}
         variant={ButtonVariant.Primary}
         color={ButtonColor.Twitter}
-        onClick={() => onClickSocial(ShareProvider.Twitter)}
+        onClick={() => onClick(ShareProvider.Twitter)}
```

Comment:
The naming is giving me headaches, but non blocking 😅

Reply:
I don't want to intentionally hurt anyone, so I'll consider renaming 😄

Reply:
I renamed to `openShareLink`, let me know if that's still awful to you 🙏